create-vm.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# This file installs multipass.
+
+if (which multipass > /dev/null)
```

Comment:
Would be better to do `multipass version`  as your conditional check instead of using `which`.

Reply:
Good idea boss! Done and tested.